tests/integration/bond_test.py

```diff
@@ -369,3 +376,30 @@ def test_bond_with_empty_ipv6_static_address(eth1_up):
         assertlib.assert_state(bond_state)
 
     assertlib.assert_absent('bond99')
+
+
+@pytest.fixture
+def linux_br0_over_vlan101(eth1_up):
+    port_state = {
+        'stp-hairpin-mode': False,
+        'stp-path-cost': 100,
+        'stp-priority': 32,
+    }
+    bridge_name = 'linux-br0'
+    port_name = eth1_up[Interface.KEY][0][Interface.NAME]
+    bridge_state = add_port_to_bridge(
+        create_bridge_subtree_state(), port_name, port_state
+    )
+    with linux_bridge(bridge_name, bridge_state) as desired_state:
+        assertlib.assert_state(desired_state)
+        yield deepcopy(desired_state)
+
+
+def test_create_linux_bridge_over_vlan_over_bond(
```

Comment:
This is not doing what its name hints.
The test is provided a bridge over eth1 and a bond over a slave (i guess eth2).
Then it creates a vlan over the bond.
No relation between the bridge and vlan/bond.